.gitignore

```diff
@@ -74,3 +74,7 @@ fastlane/screenshots
 
 # Storybook build files
 storybook-static
+
+# GitHub GPG Keys
+.github/workflows/OSBotify-private-key.asc
+*.asc
```

Comment:
FYI - I verified via `find . -name "*.asc"` that there are no other keys that would be unintentionally ignored via this.

Reply:
Just wondering if `.github/workflows/OSBotify-private-key.asc` is already included in `*.asc `

Reply:
It _should_ be yes, but this is somewhat a safety net just in case we ever change the name of the key. I could remove `.github/workflows/OSBotify-private-key.asc` if it's too redundant.

Reply:
I guess I was double checking that it was intentional, I'm ok if you want to leave it there to be more explicit as it doesn't cause any problem :P, changing my comment to NAB.

Reply:
I think it's ok to leave it